src/status_update.rs

```diff
@@ -1,9 +1,44 @@
 use super::{u2ftypes, Pin};
+use crate::ctap2::commands::get_info::AuthenticatorInfo;
+use serde::{Deserialize, Serialize as DeriveSer, Serializer};
```

Comment:
No, this is intentional / needed.
In the current draft of the about-page, `InteractiveRequest` needs to be deserialized from JSON, and the plan is to serialize `StatusPinUv` to JSON as well, so that the about-page can display appropriate error-messages (e.g. when doing "change PIN").
This could of course also be solved by repackaging everything, but IHMO having de/serialize implemented makes things much easier to handle.

`StatusUpdate` doesn't need to be serialized anymore, as per #259

Reply:
OK, we'll leave it for now.